web/src/basic/Dashboard.js

```diff
@@ -42,6 +49,39 @@ const Dashboard = (props) => {
     });
   }, [props.owner]);
 
+  const handleTourChange = () => {
+    setIsTourVisible(Setting.getTourVisible());
+  };
+
+  const setIsTourToLocal = () => {
+    Setting.setIsTourVisible(false);
+    setIsTourVisible(false);
+  };
+
+  const handleTourComplete = () => {
+    if (nextPathName !== "") {
+      props.history.push("/" + nextPathName);
+      Setting.setIsTourVisible(true);
+    }
+  };
+
+  const getSteps = () => {
+    const steps = Setting.TourObj["home"];
+    steps.map((item, index) => {
+      if (index === 2) {
+        item.target = () => document.getElementById("echarts-chart");
+      } else if (index === 1) {
+        item.target = () => document.getElementById("statistic");
+      }
+      if ("nextButtonProps" in item) {
+        item.nextButtonProps.children = nextPathName !== "" ?
+          "Go to \"" + nextPathName.charAt(0).toUpperCase() + nextPathName.slice(1) + " List\""
```

Comment:
Define util function for `"Go to \"" + nextPathName.charAt(0).toUpperCase() + nextPathName.slice(1) + " List\""`